db/migrate/20190304192641_add_provider_guids.rb

```diff
@@ -6,6 +6,10 @@ class Provider < ActiveRecord::Base
   end
 
   def up
-    Provider.in_my_region.where(:guid => nil).each { |provider| provider.update_attributes!(:guid => SecureRandom.uuid) }
+    say_with_time("Update provider guid values") do
+      Provider.in_my_region.where(:guid => nil).each do |provider|
+        provider.update_attributes!(:guid => SecureRandom.uuid)
```

Comment:
FYI, I think update_attributes is deprecated in rails 5.2.  That's going to be fun.